warehouse/query-core/src/test/java/datawave/query/testframework/AbstractFunctionalQuery.java

```diff
@@ -465,4 +500,30 @@ protected void ivaratorConfig(final int hdfsLocations, final boolean fst) throws
             this.logic.setIvaratorCacheBaseURIs(uriList);
         }
     }
+    
+    /**
+     * assertQuery is almost the same as Assert.assertEquals except that it will allow for different orderings of the terms within an AND or and OR.
+     * 
+     * @param expected
+     *            The expected query
+     * @param query
+     *            The query being tested
+     */
+    protected void assertPlanEquals(String expected, String query) throws ParseException {
```

Comment:
doesn't this belong in a test class?

Reply:
This is the base class that all test classes that use the test framework extend. It needs to be available to any test class.